chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -479,22 +479,6 @@
             </job-data-map>
         </job>
 
-        <!-- Real World Testing Eligibility Job -->
```

Comment:
Deleting a job from the jobs.xml isn't enough. We also need to put one of the "delete jobs" commands in the pre-processing commands section at the top to remove the job from the system